src/MSGEQ7.hpp

```diff
@@ -116,13 +116,8 @@ read(void){
 
 		// read pins with 8 bit resolution
 		frequency f = (frequency){
-#ifdef MSGEQ7_10BIT
-			analogRead(firstAnalogPin),
-				analogRead(analogPins)...
-#else
-			MSGEQ7_data_t(analogRead(firstAnalogPin) >> 2),
-				MSGEQ7_data_t(analogRead(analogPins) >> 2)...
-#endif
+			getAnalogReadPin(firstAnalogPin),
+			getAnalogReadPin(analogPins)...
```

Comment:
Why did you remove the 8 bit variant?

Reply:
I removed the 8 Bit variant because it's already set in the getAnalogReadPin if MSGEQ7_10BIT or MSGEQ7_12BIT is not defined.